rmarkdown_tutorial.Rmd

```diff
@@ -0,0 +1,220 @@
+- **Reproducible Research**\
+· You can use `Knit` to rerun the code in an R Markdown file to reproduce your work and export the results to be a well-formatted report.\
+- **Dynamic Documents**\
+· There are a lot of ways to export your report. The formats include html, pdf, MS Word, or RTF documents; html or pdf based slides, Notebooks, and more.\
+
+
+### 1.2 Workflow
+(1) **Open a new .Rmd file** at File ▶ New File ▶ R Markdown. \
+(2) **Write document** by editing template\
+(3) **Knit document** to create report; use knit button or
+**render()** to knit\
+(4) **Preview Output** in IDE window\
+(5) **Publish** (optional) to web server\
+(6) **Examine build log** in R Markdown console\
+(7) **Use output file** that is saved along side .Rmd
+![Caption](resources/rmarkdown_tutorial/rmarkdownflow.png)
+
+## 2. Getting started
+
+### 2.1. Install the package
+You can use the following command to install the required library.
+```{r, eval=FALSE}
+install.packages('rmarkdown')
+```
+
+
+### 2.2. Open file
+You can create a new file or open existed from the directory that you choose.
+
+### 2.3. output format
+The Rmarkdown can render any Rmd file into a format that Rmarkdown supports. For example, the code below renders OutputExample.Rmd to a Microsoft Word document. 
+```{r results='hide'}
+library(rmarkdown)
+render("OutputExample.Rmd", output_format = "word_document")
+```
+
+Following is a table of all the formats that you can choose:
+**Output value table**
+
+| output value | creates |
+|--------------|---------|
+| html_document| html|
+| pdf_document |pdf (requires Tex )|
+| word_document| Microsof Word (.docx)|
+| odt_document |OpenDocument Text|
+| rtf_document |Rich Text Format|
+| md_document  |Markdown|
+| github_document| Github compatible markdown|
+| ioslides_presentation| ioslides HTML slides|
+| slidy_presentation| slidy HTML slides|
+| beamer_presentation| Beamer pdf slides (requires Tex) |
+
+You can choose to render to the format that you want by clicking the dropdown menu beside the knit button:
+![Caption](resources/rmarkdown_tutorial/output.png)
+
+## 3. Markdown syntax
+
+Rmarkdown has lots of fancy syntax so that you can produce an ordered and beautiful document.\
+We will give the syntax here that we regularly use a lot.
+
+* **plain text**    
+plain text
+
+* **italics and bold**  
+```{}
+*italics* and **bold**
+```
+
+*italics* and **bold**  
+
+
+* **list**  
+```{}
+* unordered list  
+   + sub-item 1  
+   + sub-item 2  
+   - sub-sub-item 1 
+```
+  
+* unordered list
+  + sub-item 1
+  + sub-item 2
+  - sub-sub-item 1 
+  
+```{}
+1. ordered list
+2. item 2  
+ i) sub-item 1  
+    A. sub-sub-item 1 
+```
+1. ordered list
+2. item 2
+  i) sub-item 1  
+    A. sub-sub-item 1 
+  
+  
+* **headers**
+```{}
+# Header1 {#anchor}
+## Header 2 {#css_id}
+### Header 3 {.css_class}
+#### Header 4
+##### Header 5
+###### Header 6 
+```
+# Header1 {#anchor}
```

Comment:
I recommend inserting a photo here. There should be no single # header other than the title.